### PR TITLE
feat: Add an intro to LangGraph + Gemini notebook with tools and workflow

### PR DESCRIPTION
**Pull request by koverholt**
# Description

This PR adds a sample notebook that uses LangGraph and Gemini to automate the process of retrieving real-time and historical exchange rates from a currency API, verifying the accuracy of the data, and generating a report summarizing the findings.

- [X] Follow the [`CONTRIBUTING` Guide](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/CONTRIBUTING.md).
- [X] You are listed as the author in your notebook or README file.
  - [X] Your account is listed in [`CODEOWNERS`](https://github.com/GoogleCloudPlatform/generative-ai/blob/main/.github/CODEOWNERS) for the file(s).
- [X] Make your Pull Request title in the <https://www.conventionalcommits.org/> specification.
- [X] Ensure the tests and linter pass (Run `nox -s format` from the repository root to format).
- [X] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: gemini/orchestration/intro_langgraph_gemini.ipynb [hunk 1]
```diff
@@ -0,0 +1,761 @@
...
+            "text/plain": [
+              "<IPython.core.display.Markdown object>"
+            ]
+          },
+          "metadata": {},
+          "output_type": "display_data"
+        }
+      ],
+      "source": [
+        "display(Markdown(s[\"report\"][\"messages\"].content))"
```
Comment (holtskinner) [inline gemini/orchestration/intro_langgraph_gemini.ipynb:745]: I got an error when running this `s is not defined`. Was `s` supposed to be defined, or should a different variable be used?

**Review by holtskinner — APPROVED** [review]
Just fix the variable, then you're good to go!

**Comment by holtskinner** [comment]
Note: Owlbot is turned off, so it won't run.

File: gemini/orchestration/intro_langgraph_gemini.ipynb (hunk 1 above)
Comment (koverholt) [reply gemini/orchestration/intro_langgraph_gemini.ipynb]: Ah, great catch. I fixed this in 17a36c8 to refer correctly to the `state` variable!

**Comment by koverholt** [comment]
Thanks for the review and for testing the notebook! 🙏

**Pull request merged**